_data/beyond_essence.yml

```diff
@@ -35,4 +35,18 @@
       link: beyond-essence/inventory/toggle-switches.md
 
     - title: Toast messages
-      link: beyond-essence/inventory/toast-messages.md
\ No newline at end of file
+      link: beyond-essence/inventory/toast-messages.md
+
```

Comment:
Suggestion for the sorting of the sidebar:

Crafting an article
Structure
Design & Formatting (what about "Look & Feel")
Internationalization